.github/workflows/udpate-generated.yml

```diff
@@ -0,0 +1,52 @@
+name: 'Update Generated Files'
+
+# This workflow is triggered on a schedule (every day at 3:15 AM UTC)
+# and can also be triggered manually from the GitHub Actions UI.
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '15 3 * * *'
+
+jobs:
+  update-and-pr:
+    name: 'Update Generated Files and Create PR'
+    runs-on: ubuntu-latest
+    container: quay.io/coreos-assembler/fcos-buildroot:testing-devel
+    steps:
+      - name: 'Checkout repository'
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: 'Mark git directory as safe'
+        run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
+        shell: bash
+
+      - name: 'Install dependencies'
+        run: ./ci/installdeps.sh
+        shell: bash
+
+      - name: 'Install pandoc'
+        run: dnf -y install pandoc
+        shell: bash
+
+      - name: 'Update generated docs'
+        run: cargo xtask update-generated
+        shell: bash
+
+      - name: 'Create Pull Request'
```

Comment:
I think this is missing a step like https://github.com/coreos/fedora-coreos-config/blob/ba564a93c463f9bd051cf3333822138f38bf0a69/.github/workflows/bump-fedora-bootc.yaml#L36 to do a diff right?

Reply:
Based on my read of how the action behaves, it will silently exit if there are no changes detected - https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#action-behaviour

> How the action behaves:
> 
>   - If there are changes (i.e. a diff exists with the checked-out base branch), the changes will be pushed to a new branch and a pull request created.
>   - If there are no changes (i.e. no diff exists with the checked-out base branch), no pull request will be created and the action exits silently.

When I tested this in my fork, it correctly created a PR that looked like https://github.com/bootc-dev/bootc/pull/1385

Hmm...I should apply those changes to my `main` branch and see if the action just no-ops.  Will report back here with results!